plenigo_plugin/wooCommerce/WC_Gateway_Plenigo.php

```diff
@@ -34,6 +33,8 @@ class WC_Gateway_Plenigo extends \WC_Payment_Gateway {
      * Holds the values to be used in the fields callbacks
      */
     private $options = null;
+    // Available Product Types
+    private $prodTypeList = array('EBOOK', 'DIGITALNEWSPAPER', 'DOWNLOAD', 'VIDEO', 'MUSIC');
```

Comment:
Yes but resuding it will require and import and requiring the File, I can consider a single static class for constants and fixed values like this.....currently distributed in all classes